electrumx/server/session.py

```diff
@@ -178,6 +179,57 @@ def _ssl_context(self):
             self._sslc.load_cert_chain(self.env.ssl_certfile, keyfile=self.env.ssl_keyfile)
         return self._sslc
 
+    def _get_bolt8_server(self):
+        from electrum import ecc
+        from electrum import logging
+        from electrum.lntransport import create_bolt8_server
```

Comment:
I think it would be good to avoid depending on electrum. Or at least, let's try not to make it a required dependency.
The watchtower functionality could be optional, and if enabled, we could then require electrum.

Re the bolt8 transport bits, we should consider splitting that functionality out of electrum into a separate package, and only depend on that here.

Regardless, electrum is currently not distributed on PyPI. Whatever we end up depending on here, we should either distribute that on PyPI, or maybe make it a git submodule.